### ISSUE TITLE
Issues with question attributes listed at question level

### ISSUE THREAD

**Issue by maziminke** [olleharstedt/MassAction#16]
I am not sure how the tool determines which question attributes to include. E.g. theoretically you can add an array filter setting for a text display question which doesn't make sense at all (tested with LS 3.17)
On the other hand, I am missing the question validation hint attribute. A column for entering a question validation exists but a column for adding the hint text is missing. Would be great to add that.

In general, maybe we should just ask at the forums which attributes are edited most often and then limit the list to these?

**Comment by olleharstedt** [comment]
The hint text might be localized. I didn't add any code to deal with multiple languages.

**Comment by maziminke** [comment]
That makes sense. Would be useful to add this since adjusting default text hints is a pretty common mass action use case.

**Comment by maziminke** [comment]
@olleharstedt Since at LS 5.x the plugin would have to be adjusted anyway to deal with the different way localized details are stored, it would make sense to also work on the localized question settings then.

Any plans to publish this for LS 5.x? I think itwould be helpful to many users. At least 3-4 of our bigger customers are using this as well.

**Comment by olleharstedt** [comment]
Yeah, definitely wanna convert some of my plugins to LS5. About time...
